docs/UserGuide.md

```diff
@@ -105,15 +105,18 @@ Note: Multiple students may share the same name.
 ```
 
 2. Student’s Contact Number:
-   - Contact number must only contain numerical digits between `0` and `9`.
+   - Student’s contact number must not be empty. 
+   - Student’s contact number must only contain numerical digits between `0` and `9`.
 ```yaml
 Note: Contact number must contain at least 3 digits. Contact number must be unique.
 ```
 
-3. Next of Kin’s Number:
-    - Next of Kin’s number must only contain numerical digits between `0` and `9`.
+3. Next of Kin’s Contact Number:
+    - Next of Kin’s contact number must not be empty.
+    - Next of Kin’s contact number must contain 8 numerical digits between `0` and `9`.
+    - Next of Kin’s contact number must begin with `6`, `8` or `9`. 
 ```yaml
-Note: Next of Kin’s number cannot be empty. It must contain at least 3 digits.
+Note: Next of Kin’s contact number cannot be empty. It must contain at least 3 digits.
```

Comment:
For all the contact numbers, i think must say exactly 8 digits instead of at least 3 digits. Also i think can remove "NOK cannot be empty since it is already included above"

Reply:
Good catch! Will update in later commit.